src/SFA.DAS.Charities.Api/nlog.config

```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:xdt="http://schemas.microsoft.com/XML-Document-Transform">
+    <extensions>
+        <add assembly="SFA.DAS.NLog.Targets.Redis" />
+    </extensions>
+    <variable name="appName" value="das-charities-api"/>
+    <variable name="simplelayout" value="${longdate} [${uppercase:${level}}] [${logger}] - ${message} ${onexception:${exception:format=tostring}}"/>
+
+    <targets>
+        <target name="Console" layout="${simplelayout}" xsi:type="Console"/>
+        <target name="Redis" appName="${appName}" environmentKeyName="Environment" connectionStringName="LoggingRedisConnectionString" includeAllProperties="true" layout="${message}" xsi:type="Redis"/>
```

Comment:
Missing field aspnet-traceidentifier . This will be useful for Kibana

`<field name="RequestId" layout="${aspnet-traceidentifier}"/>`